domene/src/main/java/no/nav/foreldrepenger/journalføring/oppgave/OppgaverTjeneste.java

```diff
@@ -164,26 +179,54 @@ public void avreserverOppgaveFor(Oppgave oppgave) {
     }
 
     @Override
-    public List<Oppgave> finnÅpneOppgaverFor(Set<String> enheter) {
-        List<Oppgave> oppgaver = new ArrayList<>();
+    public List<Oppgave> finnÅpneOppgaverFiltrert() {
+        var saksbehandlerEnheter = losEnheterCachedTjeneste.hentLosEnheterFor(KontekstHolder.getKontekst().getUid())
+            .stream()
+            .map(TilhørendeEnhetDto::enhetsnummer)
+            .collect(Collectors.toUnmodifiableSet());
 
-        if (enheter == null || enheter.isEmpty()) {
-            finnOppgaver(null, oppgaver);
-        } else {
-            enheter.forEach(enhet -> finnOppgaver(enhet, oppgaver));
+        if (saksbehandlerEnheter.isEmpty()) {
+            return List.of();
         }
-        return oppgaver.stream()
+
+        return finnAlleOppgaver().stream()
+            .filter(oppgave -> !NK_ENHET_ID.equals(oppgave.tildeltEnhetsnr())) // Klager går gjennom gosys
+            .filter(ikkeSkjermingenhetsoppgave().or(saksbehandlerHarTilgangTilSkjermingenheten(saksbehandlerEnheter))) // må ha tilgang til Spesialenheten
+            .filter(brukerUkjentEllerTilgjengeligIPdl())
```

Comment:
Jeg tenker det holder med en av sjekkene - enten via LOS eller via PDL.
Dette gjelder oppgave-listen og der viser vi ikke så mye informasjon annet enn FNR.
Når man prøver åpne oppgave/dokument så vil andre sjekker slå til i hent-bruker og SAF

Reply:
Ja, bare det som er litt dumt er at om man får tilgangsnekt så havner man på en generelle side utenfor konteksten. Det beste er om man bare får fjernet alt som man ikke har tilgang til.

Er det slikt at vi får ikke noe feil fra PDL på K7 beskyttelse?

Reply:
PDL er catch-all - du får feil hvis du ikke har tilgang til k6, k7, egenansatt. K7 er veldig sjeldent hos oss.

Reply:
Ja men du sier - Den mot PDL er catchAll og fjerner behovet for 1 integrasjon - men da kan noen K7-oppgaver bli liggende usynlige.

Hvorfor blir de usynlige? Kallet mot PDL vil nok ikke feiler for saksbehandlere med tilgang til K7 - de må bare være klar over at K6 / K7 / Egen Ansatt saksbehandlere må gjennom Journalføring av og til også.